.github/workflows/ci.yml

```diff
@@ -5,11 +5,13 @@ on:
     paths-ignore:
       - "docs/**"
       - "**.md"
+      - "vercel/**"
```

Comment:
This folder may need to be added to the `exclude` section of the `Cargo.toml` file.